packages/app/android/app/src/main/AndroidManifest.xml

```diff
@@ -3,8 +3,7 @@
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.INTERNET" />
     <application
-        android:label="Meli"
-        android:name="${applicationName}"
```

Comment:
to my understanding, this isn't necessary and no `applicationName` placeholder was being specified by our build configuration (unless there's some hidden flutter magic that's defining this)

https://developer.android.com/guide/topics/manifest/application-element#nm